README.md

```diff
@@ -171,16 +173,33 @@ mstorage.list(){result, error in
         // do something
     }
 }
-// You can also use a Dictionary object for listing options as follows.
-// The available options are "limit", "after" and "before".
+```
+
+* With options
+
+You can also use a `Dictionary` object for listing options as follows.
+The available options are `limit`, `after` and `before`.
+
+```swift
 mstorage.list(["limit": "25", "after": "<media_id>"]){result, error in
     // do something
 }
 ```
 
-### Delete
+* Search
+
+Search media ids by user metadata
+
+```swift
+mstorage.list(["limit": "25", "after": "<media_id>",
```

Comment:
I think it's better that there is example to know how to combine listing options (such as "limit") and searching option ("filter").
In media-storage-js, I think the reason of search example doesn't include "limit" key is it's not available in [v1.1.1 of REST API](https://github.com/ricohapi/media-storage-rest/blob/v1.1.1/media.md#SearchMedia) and is not reflected v1.2.0 changes.
So I think we should be better to have combined example.